packages/astro/test/fixtures/legacy-build/src/pages/index.astro

```diff
@@ -1,11 +1,7 @@
 ---
-import imgUrl from '../images/penguin.jpg';
```

Comment:
I stripped this down to remove things that were undefined/not-well-supported before the static build came along.